docs/DeveloperGuide.md

```diff
@@ -271,21 +271,46 @@ _{Explain here how the data archiving feature will be implemented}_
+
+
+| Priority      | As a/an           | I want to …                                                      | So that I can…                                                     |
+|---------------|-------------------|------------------------------------------------------------------|--------------------------------------------------------------------|
+| **Must-Have** | Novice user       | add new contact information (phone-number, email)                | keep the contact information in one place                          |
+| **Must-Have** | Novice user       | view contact information                                         | view the contact information of my student, Peter                  |
+| **Must-Have** | Novice user       | delete existing contact                                          | ensure all contacts stored are relevant                            |
+| **Must-Have** | Any user          | have my data automatically saved                                 | access my information seamlessly the next time I open the software |
+| Nice-to-Have  | New user          | view the help guide easily                                       | learn how to use the app                                           |
+| Nice-to-Have  | Novice user       | edit the existing contact of Peter                               | correct the contact typo                                           |
+| Nice-to-Have  | Any user          | filter the students by lesson day and time                       | find all students in a class at once                               |
+| Nice-to-Have  | Any user          | sort students by last name                                       | have the contacts sorted                                           |
+| Nice-to-Have  | Any user          | group students by class timings                                  | send class cancellations out quicker                               |
+| Nice-to-Have  | Any user          | find Peter by his last name                                      | inform Peter of homework-related matters                           |
+| Nice-to-Have  | Any user          | filter the students by subjects                                  | view the students taking the subject                               |
+| Nice-to-Have  | Any user          | edit timing of lessons of students                               | allow students to change their lesson timing to fit their schedule |
+| Nice-to-Have  | Any user          | find Peter's parents' contact info by Peter's name               | retrieve parents' contact information quicker                      |
+| Nice-to-Have  | Elderly user      | zoom in on the smaller texts                                     | see the contact information easily                                 |
+| Nice-to-Have  | Intermediate user | tag certain contacts                                             | know the contacts' roles (e.g., admin)                             |
```

Comment:
I think we need to change the benefit or function so that it is relevant to one another.

Maybe the benefit could be so that I can assign attributes to the contact?